### ISSUE TITLE
@BindType as a repeatable annotation in Kotlin 1.6.0

### ISSUE THREAD

**Issue by jakoss** [mars885/hilt-binder#42]
If i understand correctly - kotlin 1.6.0 is lifting the restriction on the `@Repeatable`, so it can be used on any retention. I think multiple binds would be a great addition to this otherwise underestimated library

**Comment by mars885** [comment]
Yeah, that's definitely something that I've been keeping my eye on and it does make a lot of sense to make the `@BindType` annotation repeatable. I'll take a look when I have some time, because I've been pretty busy lately. Thanks for the heads up.
